### PR TITLE
Add  chat_heads example with new api

### PR DESCRIPTION
**Pull request by enterteg**
Need to be tested on Android.

### CODE REVIEW

File: android/lib/src/main/java/com/swmansion/gesturehandler/TapGestureHandler.kt
Comment (enterteg) [inline android/lib/src/main/java/com/swmansion/gesturehandler/TapGestureHandler.kt:159]: https://github.com/software-mansion/react-native-gesture-handler/pull/1609

copied fix for taps not working on ios

File: examples/Example/src/new_api/chat_heads/index.tsx
Comment (jakub-gonet) [inline examples/Example/src/new_api/chat_heads/index.tsx:49]: Why `useAnimatedReaction` instead of `useDerivedValue`?

File: examples/Example/src/new_api/chat_heads/index.tsx
Comment (jakub-gonet) [inline examples/Example/src/new_api/chat_heads/index.tsx:96]:
```suggestion
    .onEnd(({ absoluteX, absoluteY }) => {
      const { height, width } = dimensions;
```

File: examples/Example/src/new_api/chat_heads/index.tsx
Comment (jakub-gonet) [inline examples/Example/src/new_api/chat_heads/index.tsx:140]:
```suggestion
  const headsComponents = CHAT_HEADS.map(({ imageUrl }, idx) => {
```

**Review by jakub-gonet — COMMENTED** [review]
Should wait until #1609 is merged.

File: examples/Example/src/new_api/chat_heads/index.tsx
Comment (enterteg) [reply examples/Example/src/new_api/chat_heads/index.tsx:49]: Yeah I forgot about it, thanks!

**Review by jakub-gonet — CHANGES_REQUESTED** [review]
`examples/Example/src/new_api/bottom_sheet/index.tsx` should be removed.

**Review by jakub-gonet — APPROVED** [review]

**Pull request merged**
